tools/data-handler/src/calculate.ts

```diff
@@ -415,12 +414,12 @@ export class Calculate {
     const firstCard = cards[0];
     await this.setCalculateProject(firstCard); // can throw
     const cardTreeFile = join(
-      Calculate.project.calculationFolder,
+      Calculate.project.paths.calculationFolder,
```

Comment:
There's a lot joins in the calculate.ts file. Would it be more clear, if all these paths were provided by the new class?

Reply:
I'll check, if we good move it there as well.